spec/lib/lighthouse/auth/client_credentials/jwt_generator_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'lighthouse/auth/client_credentials/jwt_generator'
+
+RSpec.describe Auth::ClientCredentials::JWTGenerator do
+  let(:public_key) { OpenSSL::PKey::RSA.new(File.read('spec/support/certificates/lhdd-fake-public.pem')) }
+
+  before do
+    allow(Auth::ClientCredentials::JWTGenerator).to receive(:build_rsa_instance).and_return(
```

Comment:
I'm wondering if it's a bad idea to stub this method here. It seems like if the real `build_rsa_instance` method was altered and returned something different (say: `false` or something) the test below would still pass. Curious what your thoughts are though

Reply:
Now that the file check/read operation has been added back to the `build_rsa_instance` method, there is really no need for the stub.  I can update the test to simply pass in the private key file.  That work?

Reply:
Yeah I think that sounds great!